community/lucene-index/src/main/java/org/neo4j/kernel/api/impl/labelscan/LuceneLabelScanStore.java

```diff
@@ -114,9 +116,22 @@ else if ( !luceneIndex.isValid() )
         }
     }
 
+    private void create() throws IOException
```

Comment:
yeah thats true, in previous indexes we have a partitions couple of levels below that where guarded anyway;
ok then it lets keep it for now like it is know; that's true - do not think we will have any problems with it